internal/storage/clickhouse.go

```diff
@@ -405,10 +405,16 @@ func addContractAddress(table, query string, contractAddress string) string {
 }
 
 func getTopicValueFormat(topic string) string {
-	toAddressHex := ethereum.HexToAddress(topic)
-	toAddressPadded := ethereum.LeftPadBytes(toAddressHex.Bytes(), 32)
-	toAddressTopic := ethereum.BytesToHash(toAddressPadded).Hex()
-	return toAddressTopic
+	if topic == "" {
+		// if there is no indexed topic, indexer stores an empty string
+		// we shouldn't pad and hexify such an argument then
+		return ""
```

Comment:
Can we change this to nil already since we're already working on this or will it blow it up too much?

Reply:
that's a separate thing I wanted to bring up, actually

saw [this](https://clickhouse.com/docs/en/cloud/bestpractices/avoid-nullable-columns) and thought that maybe we can actually leave it as is, but change the columns to not allow NULLs?

Reply:
yeah, sounds good. Doesn't really matter '' or null for our use case. We can change the schema and then apply it manually

Reply:
since we anyway seem to have no benefit of NULL-ability in these columns and bloom filters are behaving the same way with negation - we can at least optimise some space and performance by completely turning it of

(and, consequently, continuing to use empty strings for topics that don't exist)